SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Reporting Security Issues
+
+The Toposware team and community take security bugs in ICE-FROST seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+
+To report a security issue, please send an email to "security at toposware.com" with as much information as possible and, if possible, steps to reproduce the problem.
+
+The Toposware team will send a response indicating the next steps in handling your report. After the initial reply to your report, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
+
+Report security bugs in third-party modules to the person or team maintaining the module.
```

Comment:
nit: maybe `dependencies` / `dependency` would be more common than `modules` / `module` ?

Reply:
What if it's not technically a dependency, but some random utility/script/template or other artifact? I like "module" because it's so generic and all-encompassing.